src/databricks/azext_databricks/tests/latest/test_databricks_scenario.py

```diff
@@ -0,0 +1,70 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+import os
+import unittest
+
+from azure_devtools.scenario_tests import AllowLargeResponse
+from azure.cli.testsdk import (ScenarioTest, ResourceGroupPreparer,
+                               JMESPathCheck, JMESPathCheckExists,
+                               NoneCheck)
+from msrestazure.tools import resource_id
+
+
+TEST_DIR = os.path.abspath(os.path.join(os.path.abspath(__file__), '..'))
+
+
+class DatabricksClientScenarioTest(ScenarioTest):
+
+    @ResourceGroupPreparer(name_prefix='cli_test_databricks')
+    def test_databricks(self, resource_group):
+
+        self.kwargs.update({
+            'workspaceName': 'my-test-workspace',
+            'subscription': '00000000-0000-0000-0000-000000000000'
+        })
+
+        self.cmd('az databricks workspace create '
+                 '--resource-group {rg} '
+                 '--name {workspaceName} '
+                 '--location "westus" '
+                 '--sku standard',
+                 checks=[JMESPathCheck('name', self.kwargs.get('workspaceName', ''))])

```

Comment:
`az databricks workspace create` has many parameters. This test case is too simple. You can specify more parameters in this test cases and add more test cases to cover the parameters and code.

Reply:
I have added more parameters in tests but not all. Let's focus on the most common cases in breadth coverage.